src/hyperion/experiment_plans/grid_detect_then_xray_centre_plan.py

```diff
@@ -101,6 +118,17 @@ def create_parameters_for_flyscan_xray_centre(
     return flyscan_xray_centre_parameters
 
 
+def create_parameters_for_panda_flyscan_xray_centre(
```

Comment:
Could: I think with some generics stuff you can merge this and `create_parameters_for_flyscan_xray_centre`

Reply:
I'll do this when I get around to doing the other generic stuff